package/MDAnalysis/core/groups.py

```diff
@@ -2894,16 +2902,15 @@ def select_atoms(self, sel, *othersel, **selgroups):
            periodic are now on by default (as with default flags)
         .. versionchanged:: 2.0.0
             Added the *smarts* selection.
+        .. versionchanged:: 2.0.0
```

Comment:
merge with the previous section? – Would make more sense from user perspective but I don't know how we've been handling multiple additions in the same version elsewhere.